src/sentry/integrations/utils/code_mapping.py

```diff
@@ -535,3 +537,29 @@ def _get_code_mapping_source_path(src_file: str, frame_filename: FrameFilename)
     if source_code_root:
         assert source_code_root.endswith("/")
     return source_code_root
+
+
+def find_roots(stack_path, source_path) -> tuple[str, str]:
```

Comment:
```suggestion
def find_roots(stack_path: str, source_path: str) -> tuple[str, str]:
```

When the full signature is typed, then mypy does a better job at typing rather than assuming `Any`.